ci-operator/step-registry/cucushift/installer/rehearse/aws/ipi/disconnected/private/cco-manual-security-token-service/efs/provision/cucushift-installer-rehearse-aws-ipi-disconnected-private-cco-manual-security-token-service-efs-provision-chain.yaml

```diff
@@ -0,0 +1,43 @@
+chain:
+  as: cucushift-installer-rehearse-aws-ipi-disconnected-private-cco-manual-security-token-service-efs-provision
+  steps:
+    - ref: aws-provision-vpc-disconnected
+    - chain: aws-provision-bastionhost
+    - chain: mirror-images-by-oc-adm
+    - ref: ipi-conf
+    - ref: ipi-conf-telemetry
+    - ref: ipi-conf-aws-custom-vpc
+    - ref: ipi-conf-manual-creds
+    - ref: ipi-conf-aws-oidc-creds-provision
+    - ref: ipi-conf-aws
+    - ref: ipi-conf-mirror
+    - ref: ipi-install-monitoringpvc
+    - ref: proxy-config-generate
+    - ref: ipi-conf-manual-creds-remove-unnecessary-creds
+    - ref: ipi-install-install-aws
+    - ref: ipi-install-times-collection
+    - ref: aws-provision-tags-for-byo-vpc
+    - ref: enable-qe-catalogsource-disconnected
+    - ref: set-sample-operator-disconnected
+    - chain: storage-conf-csi-optional-aws-efs
```

Comment:
If `storage-conf-csi-optional-aws-efs` chain is the only new step based on `cucushift-installer-rehearse-aws-ipi-disconnected-private-cco-manual-security-token-service-provision`, you can reference `cucushift-installer-rehearse-aws-ipi-disconnected-private-cco-manual-security-token-service-provision` here instead of the above multiple lines.

Reply:
Thanks! Good point, sorry I missed it. updated.